perllib/FixMyStreet/App/Controller/Questionnaire.pm

```diff
@@ -234,6 +234,8 @@ sub process_questionnaire : Private {
 
     map { $c->stash->{$_} = $c->get_param($_) || '' } qw(been_fixed reported another update);
 
+    $c->stash->{update} = Utils::cleanup_text($c->stash->{update});
```

Comment:
The other update handling code includes `{ allow_multiline => 1 }` - should it be here too?